src/API.cc

```diff
@@ -0,0 +1,146 @@
+#include "API.h"
+
+#include "snowcrash.h"
+
+#include "refract/Element.h"
+#include "refract/FilterVisitor.h"
+#include "refract/Query.h"
+
+
+#include "SerializeResult.h" // FIXME: remove - actualy required by WrapParseResultRefract()
+#include "Serialize.h" // FIXME: remove - actualy required by WrapperOptions
+#include "RefractDataStructure.h" // FIXME: remove - required by SerializeRefract()
+
+#include "sos.h" // FIXME: remove sos dependency
+#include "sosJSON.h"
+#include "sosYAML.h"
+
+#include "ConversionContext.h"
+
+#include <string.h>
+
+DRAFTER_API int drafter_parse_blueprint_to(const char* source,
+                               const char ** out,
+                               drafter_options options) {
+
+    drafter_result* result = nullptr;
+    *out = nullptr;
+
+    int ret = drafter_parse_blueprint(source, &result);
+
+    if (!result) {
+        // FIXME: there is no parse result
+        return -1;
+    }
+
+    *out = drafter_serialize(result, options);
+
+    free_drafter_result(result);
+
+    return ret;
+}
+
+namespace sc = snowcrash;
+
+/* Parse API Bleuprint and return result, which is a opaque handle for
+ * later use*/
+DRAFTER_API int drafter_parse_blueprint(const char* source, drafter_result** out) {
+
+    if (!source) {
```

Comment:
Yes, I remember, we spoke about it, but I did not touch code since this time because of support